indica/models/magnetic_recon.py

```diff
@@ -0,0 +1,87 @@
+import xarray as xr
+import matplotlib.pyplot as plt
+
+from indica.models.abstractdiagnostic import DiagnosticModel
+from indica.models.plasma import example_run as example_plasma
+from indica.readers.available_quantities import AVAILABLE_QUANTITIES
+
+
+class MagneticRecon(DiagnosticModel):
```

Comment:
I would change the name to **EquilibriumReconstruction**.
We can envisage it to calculate all the relevant equilibrium quantities including, in the far future, the internal equilibrium reconstruction (hence the name) using the EFIT boundary, similar to ASTRA or TRANSP.